cli/ops/process.rs

```diff
@@ -33,12 +33,12 @@ fn clone_file(
   })
 }
 
-fn subprocess_stdio_map(s: &str) -> std::process::Stdio {
+fn subprocess_stdio_map(s: &str) -> Option<std::process::Stdio> {
   match s {
-    "inherit" => std::process::Stdio::inherit(),
-    "piped" => std::process::Stdio::piped(),
-    "null" => std::process::Stdio::null(),
-    _ => unreachable!(),
+    "inherit" => Some(std::process::Stdio::inherit()),
+    "piped" => Some(std::process::Stdio::piped()),
+    "null" => Some(std::process::Stdio::null()),
+    _ => None,
   }
```

Comment:
Rather than returning `Option<Stdio>` here return `Result<Stdio, ErrBox>`, do reduce the amount of code below.